CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   - drag: clauses move intuitively in `clojure.test/are`
   - drag: top-level forms can be dragged #891
   - Improve completion performance for most cases, reducing time to compute clj/cljs core symbols.
+  - completion: suggest functions defined in Clojure 1.10 and 1.11
```

Comment:
We already suggest those functions if user using clojure 1.10/1.11, it's just that this PR makes them available in its cache, so maybe rename this to `completion: Make Clojure 1.10 and 1.11 new functions available on completion cache`

Reply:
Will you double check that's true? For example, I don't get `update-vals` as a completion option until this patch is applied.

Reply:
Yeah, you are right, there is a chance we have a bug here or on kondo or some cache